.github/workflows/build.yml

```diff
@@ -73,30 +57,28 @@ jobs:
     - name: Print hashes
       if: matrix.os == 'ubuntu-latest'
       run: |
+        pip install -U wheel
         make generate-manifest
         python setup.py sdist
         mv dist/psutil*.tar.gz wheelhouse/
         python scripts/internal/print_hashes.py wheelhouse/
 
-  # Linux + macOS + Python 2
+  # Linux + macOS + Python 2.7 & 3.5
   linux-macos-py2:
-    name: ${{ matrix.os }}-py2
+    name: ${{ matrix.os }}-py27-py35
     runs-on: ${{ matrix.os }}
     timeout-minutes: 20
     strategy:
       fail-fast: false
       matrix:
         os: [ubuntu-latest, macos-12]
-        include:
-          - {name: Linux, python: '3.9', os: ubuntu-latest}
     env:
-      CIBW_ARCHS_LINUX: 'x86_64 i686'
       CIBW_TEST_COMMAND:
         PYTHONWARNINGS=always PYTHONUNBUFFERED=1 PSUTIL_DEBUG=1 python {project}/psutil/tests/runner.py &&
         PYTHONWARNINGS=always PYTHONUNBUFFERED=1 PSUTIL_DEBUG=1 python {project}/psutil/tests/test_memleaks.py
       CIBW_TEST_EXTRAS: test
-      CIBW_BUILD: 'cp27-*'
-      CIBW_SKIP: '*-musllinux_*'
```

Comment:
Please let's keep this PR related to macOS only. There is another PR related to Musl Linux and I haven't decided yet on whether to provide wheels for that platform.

Reply:
cibuildwheel 1.12.0 will never build musllinux & there are no images for musllinux cp27.
I reverted the change adding cp 3.5 on linux.

Reply:
remove cp 3.11 as well.